mlflow/sklearn.py

```diff
@@ -103,6 +131,8 @@ def _load_model_from_local_file(path):
     assert "sklearn" in model.flavors
     params = model.flavors["sklearn"]
     with open(os.path.join(path, params["pickled_model"]), "rb") as f:
+        # Models serialized with Cloudpickle can be deserialized using Pickle, so there
```

Comment:
are there any disadvantages of doing this? if so ... let's do the right thing. write pickling format to filename. in future, that will also enable using different serialization formats.

Reply:
Good question! `Cloudpickle.load()` is just a redefinition of `pickle.load()`: https://github.com/cloudpipe/cloudpickle/blob/51be0f98e76a3bfcca2333d6519f336e508d50a3/cloudpickle/cloudpickle.py#L938.

I've updated the comment to clarify this. We're also currently saving the serialization format in the `save_model` function; this will allow us to modify the behavior if cloudpickle changes in the future. Avoiding the switch-case at this time is beneficial because it means that models saved in cloudpickle format can be deserialized without the cloudpickle library dependency; existing users will not have to install an additional library to ensure that they can load sklearn models.